src/components/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Paper, Container } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+const styles = (theme) => ({
+  content: {
+    flexGrow: 1,
+  },
+  description: {
+    padding: theme.spacing(2),
+    textAlign: 'left',
+    color: theme.palette.text.secondary,
+  },
+  image: {
+    maxHeight: '800px',
+    width: '70%',
+  },
+  blogSide: {
+    marginTop: '120px',
+  },
+});
+
+function BlogPost(props) {
+  const { classes } = props;
+
+  return (
+    <Container>
+      <Grid container spacing={3}>
+        <Grid className={classes.blogSide} item xs={3}>
+          <Grid>{props.blogSideDate}</Grid>
```

Comment:
Destructure all these props here so that we don't see props everywhere in the jsx.